src/usdb_syncer/addons.py

```diff
@@ -9,7 +9,17 @@
 
 
 def load_all() -> None:
-    sys.path.append(str(utils.AppPaths.addons))
-    for _, mod, is_pkg in pkgutil.iter_modules([str(utils.AppPaths.addons)]):
+    addons_path = utils.AppPaths.addons
+    sys.path.append(addons_path.as_posix())
```

Comment:
Though modern Windows systems support posix paths, one should actually use str(path).